script/main.py

```diff
@@ -248,33 +248,45 @@ def average_age(conn, gender):
     print('Overall average age: {value:.2f} years'.format(value = average_age_overall))
 
 
-def most_popular_cities(conn, number):
+def all_cities(conn, number):
```

Comment:
Make it simpler. do not create new functions.
Just let function return list of strings which should be print and then print it in main.
In test assert that function returns correct number of strings to print